src/streamingCalls/streaming.ts

```diff
@@ -134,6 +159,7 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
         message: 'OK',
```

Comment:
Should this be also put inside `if (!responseHasSent)`?

Reply:
I think it doesn't matter. The stream event sequence is metadata first, then status is the last event to be emit.
We only care when the `status` received, we don't emit the `response` twice.
@murgatroid99 WDYT

Reply:
I agree.